roofit/roofitcore/inc/RooMCMC.h

```diff
@@ -0,0 +1,102 @@
+/*****************************************************************************
+ * Project: RooFit                                                           *
+ * Package: RooFitCore                                                       *
+ *    File: $Id: RooMCMC.h,v 1.00 2017/14/11 11:13:42
+
+ * Author:                                                                  *
+ *   OD, Oliver Dahme, University of Zurich, o.dahme@cern.ch      *
+ *
+ *****************************************************************************/
+#ifndef ROO_MCMC
+#define ROO_MCMC
+
+#include "TObject.h"
+#include <vector>
+#include "TFile.h"
+
+class RooAbsReal ;
+class RooFitResult ;
+class RooArgList ;
+class RooAbsArg ;
+class TVirtualFitter ;
+class TH2D ;
+class TH1F ;
+class RooPlot ;
+class TFile;
+class TMultiGraph;
+class TGraph;
+
+class RooMCMC : public TObject {
```

Comment:
Would it make sense to have a more verbose name (for example be more explicit about " Markov Chain")?

Reply:
How about RooMarkov or RooMarkovChain or RooMCMarkovChain?

And sorry for the dumb questions:
What exactly is the PR announcement?
do you mean comment each function int the header file or the cxx file?

Reply:
> What exactly is the PR announcement?

The first 'post' in this conversation :)

Reply:
Ah ok I understand I will add it as the documentation of the class right away

Reply:
> How about RooMarkov or RooMarkovChain or RooMCMarkovChain?

I would pick RooMCMarkovChain

Reply:
good name is changed. Does anybody else has suggestions for names?